snappy/services.go

```diff
@@ -153,12 +166,11 @@ func addPackageServices(m *snapYaml, baseDir string, inhibitHooks bool, inter in
 		// inhibitHooks mode
 		//
 		// *but* always run enable (which just sets a symlink)
-		serviceName := filepath.Base(generateServiceFileName(m, app))
+		serviceName := filepath.Base(generateServiceFileName(app))
 		sysd := systemd.New(dirs.GlobalRootDir, inter)
-		if !inhibitHooks {
```

Comment:
If we remove `inhibitHooks` here, we should probably remove the above comment too about inhibitHook mode.